frontend/templates/views/policies.j2

```diff
@@ -0,0 +1,17 @@
+{% extends '/layouts/default.j2' %}
+
+{% block styles %}
+{{ super() }}
+
+{% endblock %}
+
+{% block main %}
+{% do doc.styles.addCssFile('css/components/templates/policies.css') %}
+<main class="ap--main ap-t-policies">
+  <section class="ap--container">
+    <div class="ap--content-wide">
+      {{ doc.html|render|safe }}
```

Comment:
Don't we have a just-content-template already?

Reply:
none that I could find that isn't loading a lot of unnecessary code.

Reply:
Alright, can we rename to `meta.j2/scss` then? 😉

Reply:
aren't they already renamed or did I miss a file?

Reply:
No all good I was just looking not accurately 🙂 Sorry!